### PR TITLE
Add 8 social profiles: Solves #98

### PR DESCRIPTION
**Pull request by kubami9**
Solves #98
**Add 8 social profiles:**
 - facebook of Damini Satya Kammakomati
 - twitter of Steven Shiau
 - twitter of Sudheesh Singanamalla
 - github of Harish Pillay
 - facebook, instagram, github and bitbucket of Justin Lee

Preview link: https://kubami9.github.io/2017.fossasia.org-km-contribute/

### CODE REVIEW

**Review by isuruAb — COMMENTED** [review]
please squash your commits

**Review by anshumanv — APPROVED** [review]
LGTM but squashing commits is appreciated.

**Comment by kubami9** [comment]
@isuruAb @anshumanv done 😉

File: index.html [hunk 1]
```diff
@@ -1748,7 +1752,11 @@ <h2>Roland Turner</h2>
                                         <div class="hover-state text-center preserve3d">
                                             <div class="social-links vertical-align">
                                                 <a href="http://justinlee.sg" target="default"><i class="icon fa fa-external-link"></i></a>
-                                                <a href="https://twitter.com/triplez82" target="default"><i class="icon social_twitter"></i></a>
+												<a href="https://twitter.com/triplez82" target="default"><i class="icon social_twitter"></i></a>
```
Comment (rupav) [inline index.html]: You need to remove extra spaces , align them as earlier. Do it for other added lines too.

**Comment by kubami9** [comment]
@rupav fixed and squashed, in edit history it may look different but when you open the file you can see it's aligned now 😉

**Review by 0x48piraj — APPROVED** [review]

**Pull request merged**